### PR TITLE
disable bitcode on try builders

### PR DESCRIPTION
**Pull request by dnfield**
bitcode makes builds take up to 10x longer.  We don't need it on try builders.

I also regenerated the config file from the main.star

@christopherfujino  - is it ok to bump xcode for the one that was missing?

@LongCatIsLooong - did you intentionally or manually edit that one?

@iskakaushik - does this look ok to you (give fuchsia ctl to all builders as a property?)

@godofredoc  - we may want to add some CI to this repo to make sure main.star can successfully run.

### CODE REVIEW

File: config/cr-buildbucket.cfg [hunk 1]
```diff
@@ -868,7 +876,7 @@ buckets: <
         cipd_package: "infra/recipe_bundles/chromium.googlesource.com/chromium/tools/build"
         cipd_version: "refs/heads/master"
         properties_j: "$build/goma:{\"use_luci_auth\":true}"
-        properties_j: "$depot_tools/osx_sdk:{\"sdk_version\":\"10e125\"}"
+        properties_j: "$depot_tools/osx_sdk:{\"sdk_version\":\"11a420a\"}"
```
Comment (dnfield) [inline config/cr-buildbucket.cfg]: @christopherfujino

**Comment by LongCatIsLooong** [comment]
I simply relanded @chinmaygarde 's PR.

File: config/cr-buildbucket.cfg (hunk 1 above)
Comment (christopherfujino) [reply config/cr-buildbucket.cfg]: LGTM

**Comment by dnfield** [comment]
I'm going to land this since it should speed up try jobs immensely.  Happy to address further feedback in a subsequent CL.

I also want to land it so I can add a CI check that main.start is valid :)

**Pull request merged**